src/Microsoft.OData.Edm/EdmModelVisitor.cs

```diff
@@ -228,6 +229,33 @@ public virtual void VisitEntityContainerElements(IEnumerable<IEdmEntityContainer
             }
         }
 
+        public virtual async Task VisitEntityContainerElementsAsync(IEnumerable<IEdmEntityContainerElement> elements)
+        {
+            foreach (IEdmEntityContainerElement element in elements)
+            {
+                switch (element.ContainerElementKind)
+                {
+                    case EdmContainerElementKind.EntitySet:
+                        await this.ProcessEntitySetAsync((IEdmEntitySet)element);

```

Comment:
Should we add `ConfigureAwait(false)` to these await calls (and all other intsances in the PR)?

Reply:
Fixed